### PR TITLE
Update on MySQL-5.7 compatibility

### PR DESCRIPTION
**Pull request by djmitche**
For bug [3421](http://trac.buildbot.net/ticket/3421)

### CODE REVIEW

**Comment by djmitche** [comment]
I verified that 5.5 allows `default_storage_engine`

File: master/docs/relnotes/index.rst [hunk 1]
```diff
@@ -26,6 +26,9 @@ Fixes
 Deprecations, Removals, and Non-Compatible Changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+* Note that Buildbot is not yet compatible with MySQL-5.7.
+  See :bb:bug:`3421` for status and to help out.
```
Comment (rutsky) [inline master/docs/relnotes/index.rst]: I believe should be just `:bug:`, not `:bb:bug:`.

Comment (djmitche) [inline master/docs/relnotes/index.rst]: fixed

**Comment by sa2ajj** [comment]
Looks like we have issues with tests...

**Comment by djmitche** [comment]
That should be better..

**Pull request merged**